src/blueapi/config.py

```diff
@@ -162,6 +162,9 @@ class OIDCConfig(BlueapiBaseModel):
     )
     client_id: str = Field(description="Client ID")
     client_audience: str = Field(description="Client Audience(s)", default="blueapi")
+    logout_redirect_endpoint: str = Field(
+        description="The oidc endpoint required to logout", default="/oauth2/sign_out"
+    )
```

Comment:
```suggestion
    logout_redirect_endpoint: str | None = Field(
        description="The oidc endpoint required to logout", default=None
    )
```

We shouldn't redirect unless we have been configured to do so- this default can go into the template, but now we won't accidentally log people out of unrelated services.